articles/migrations/guides/legacy-lock-api-deprecation.md

```diff
@@ -0,0 +1,195 @@
+
+## Other considerations
+
+A few specific items are changing with the new versions / endpoints. If you use any of the following features, you might want to take a look at the corresponding section below.
+
+* [How to get user info](#how-to-get-user-info)
+* [How to check for an existing session](#session-management)
+* [How to log users out](#how-to-log-users-out)
+
+### How to get user info
+
+Once a user has been authenticated, an application may wish to retrieve information about a user. The recommended way to do this now is via the [/userinfo](/api/authentication#get-user-info) endpoint. If you are using Auth0.js v8 or v9 and using the [userInfo()](/libraries/auth0js/v9#extract-the-authresult-and-get-user-info) method, you already have made this change.
+
+For customers who are using the [/tokeninfo](/api/authentication#get-token-info) endpoint, this endpoint is being replaced with the /userinfo endpoint. Customers should migrate to use the /userinfo endpoint instead of /tokeninfo. The /userinfo endpoint is the only one that will be maintained going forward and is the only endpoint for user information that is supported with the [custom domains](/custom-domains) feature.
+
+As explained in the /userinfo endpoint docs entry, /userinfo obtains information using the Management API and therefore requires an [Access Token](/tokens/access-token#how-to-get-an-access-token) (obtained during login) instead of the [ID Token](/tokens/id-token) used by /tokeninfo.
+
+Note also that the [/userinfo response](/api-auth/tutorials/adoption/scope-custom-claims) may vary based on scopes requested and the value of the [OIDC Conformant](/api-auth/tutorials/adoption/oidc-conformant) setting in the [Dashboard](${manage_url}) under **Applications > (Your Application) > Settings > Advanced Settings**. In that case, application code might need adjusted to handle the slightly altered response format.
+
+### Session management
+
+#### Single Page Applications
+
+If a user navigates to a new page in a Single Page Application, your application may wish to check if a user already has an existing session. In order to do this, you may have directly called the /ssodata endpoint or utilized the `getSSOData()` function in auth0.js v8 or prior. The /ssodata endpoint is deprecated and will be removed from service on **July 16, 2018**. The `getSSOData()` function will continue to work, but will behave differently, and in most cases, can be replaced with use of `checkSession()`.
+
+::: note
+The `getSSOData()` and `checkSession()` functions should only be used from a Single Page Application
+:::
+
+##### checkSession
+
+* The Auth0.js `checkSession()` function only checks whether or not a user has an existing session in Auth0.
+* Invoking the `checkSession()` function will trigger an /authorize call, which will in turn result in the execution of [rules](/rules).
```

Comment:
I'd say "the Auth0.js checkSession() can be used to check whether or not a user has an existing session in Auth0"

It does more things, like returning fresh tokens and returning tokens for different APIs